numpy/distutils/ccompiler_opt.py

```diff
@@ -243,7 +258,8 @@ class _Config:
         ),
         AVX512_SKX = dict(
             interest=42, implies="AVX512CD", group="AVX512VL AVX512BW AVX512DQ",
-            detect="AVX512_SKX", implies_detect=False
+            detect="AVX512_SKX", implies_detect=False,
+            extra_checks="AVX512BW_MASK"
```

Comment:
Shouldn't this be 
```suggestion
            extra_checks=["AVX512BW_MASK"]
```

If not, where is the conversion from `str` to `list` occurring?

Reply:
both acceptable `extra_checks=['x', 'xx']` or`extra_checks='x xx'`

> If not, where is the conversion from str to list occurring?

https://github.com/numpy/numpy/blob/97d2db483fc0ffd46f38d0e1c39d5fc001e33197/numpy/distutils/ccompiler_opt.py#L1139-L1144

Reply:
Thanks, missed that because the line doing the splitting wasn't part of the diff